CONTRIBUTING.md

```diff
@@ -1,126 +1,113 @@
-Please read through and follow [Chris Beams's - How to write a git commit message](http://chris.beams.io/posts/git-commit/).
-
-1. Run `git log --oneline master..your-branch-name | wc -l`
-   to see how many commits there are on your branch.
-1. Run `git rebase -i HEAD~#` where # is the number
-   of commits you have done on your branch.
-1. Use the interactive rebase to edit your history.
-   Unless you have good reason to keep more than one
-   commit, it is best to mark the first commit
-   with 'r' (reword) and the others with 's' (squash).
-   This lets you keep the first commit only, but change
-   the message.
-
-If your commit resolves some issue then please refer the issue.
-This will get issue closed automatically upon merge.
-Commit message example:
+
+# Ваші внески
+
+Ми б хотіли заохочувати усіх студентів та менторів долучатися до розробки цього
+курсу. Матеріали тут здебільшого розробляються та керуються командою менторів,
+але ми визнаємо що ми можемо робити помилки, або пропускати деякі речі. Ми
+створили деякі правила, які повинні допомогти вам почати робити свій внесок.
+
+Дякуємо усім за допомогу в покращенні курсу для усіх!
+
+
+## Питання, занепокоєння та проблеми
+
+Якщо у вас виникло запитання або проблема, і ви вважаєте що така ж ситуація
+могла б виникнути і у інших, спочатку прочитайте це.
+
+Трекер проблем Гітхабу це канал якому ми надаємо перевагу при роботі з такими
+питаннями та проблемами. Він тримає питання публічними та надає можливість іншим
+долучатись з потенційними рішеннями, також він дбає про те щоб все було
+задокументовано на майбутнє. Ця система створена не для особистих запитань, або
+проблем що не підходять для публічної бесіди, у таких ситуаціях ми пропонуємо
+звертатись до когось з команди менторів, що відповідає за цей курс.
+
+Перед створенням нової проблеми, вокористайте пошук GitHub (або продивіться
+[відкриті проблеми](https://issues) або
+[закриті проблеми](https://issues?q=is%3Aissue+is%3Aclosed)) щоб переконатись,
+що подібне запитання не ставив до вас ніхто. Якщо не знайшли - вперед,
+створювайте нову. Коли створюєте нову проблему, будь ласка, приділіть час на
+вибір точної та легко впізнаваної теми, після чого надайте якомога більше
+деталей в основний опис.
+
+Приклади поганих тем:
+* _Проблема з матеріалами на поточній фазі_
+* _Питання про курс_
+* _Просто думка..._
+* _Не можу виконати завдання_
+
+
+Приклади гарних тем:
+* _Завдання "Вступ в JS": не працює посилання на додатковий матеріал_
+* _Завдання здати вправу "Додаток Pizza" 30го лютого некоректне_
+* _"Об'єктно-орієнтований JavaScript" не має репозиторію з вихідним кодом_
+
+## Поправки та додатки
+
+Ми також були б вдячні вам за ваші внески до матеріалів. Помітили помилку, або
+хотіли б додати ресурс? Використовуйте pull requests. Якщо хочете щоб ваш пул
+реквест був опрацьований якомога скоріше, будь ласка продивіться наступні
+правила. Як тільки пул реквест буде створено, його можуть переглянути студенти
```

Comment:
Can you provide me with some info: why phrase `pull request` is written in ukrainian but in line 48 in english?